docs/proposals/35-observability2.0-victoria-logs.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Migrating from Vali to VictoriaLogs
+gep-number: 35
+creation-date: 2025-10-13
+status: implementable
+authors:
+- "@nickytd"
+- "@rrhubenov"
+reviewers:
+- "@plkokanov"
+---
+
+# GEP-35: Migrating from Vali to VictoriaLogs
+
+## Table of Contents
+
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+- [Alternatives](#alternatives)
+
+## Summary
+
+This proposal introduces the deployment of [VictoriaLogs](https://github.com/VictoriaMetrics/VictoriaLogs) which will act as the replacement for [Vali](https://github.com/credativ/vali) in the Control Plane of all Shoot clusters that have the observability stack enabled, as well as in Seed clusters and Garden clusters.
+After the work that has been done on [GEP-34](https://github.com/gardener/gardener/pull/11861), we are in a favourable position to easily switch our storage layer for the clusters' log signals.
+
+A new operator is introduced that will be deployed to the `garden` namespace of seed & garden clusters.
+This operator will manage the deployment of the new `VictoriaLogs` instances to the `garden` namespace of seed & garden clusters, as well as to the Control Plane namespace of Shoot clusters via the `VLSingle` CustomResource.
+
+Since we'd like to make the transition from `Vali` to `VictoriaLogs` as painless as possible, this proposal also introduces multiple ways to migrate in already running landscapes without losing data that has accumulated with the existing `Vali` instances.
+
+## Motivation
+
+Since version v2.3.0, [Loki](https://github.com/grafana/loki) has switched from the Apache-2.0 license to a significantly more restrictive AGPLv3 licene.
```

Comment:
```suggestion
Since version v2.3.0, [Loki](https://github.com/grafana/loki) has switched from the Apache-2.0 license to a significantly more restrictive AGPLv3 license.
```